packages/core/src/db/models/Users.ts

```
@@ -727,33 +730,68 @@ export const loadUserClass = (models: IModels) => {
         ],
-        isActive: true
+        isActive: true,
       });
 
       if (!user || !user.password) {
         // user with provided email not found
-        throw new Error("Invalid login");
+        throw new Error('Invalid login');
       }
 
       const valid = await this.comparePassword(password, user.password);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)